integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/PerformanceSteps.java

```diff
@@ -21,39 +26,79 @@
 public class PerformanceSteps {
 
     private final TestContext context;
+    private final Map<String, String> benchmarkRuns = new HashMap<>();
```

Comment:
This can be stored in some context, possibly something like `PerfTestContext`?

Reply:
yes, created a new class, please have a look